opm/material/densead/Evaluation.hpp

```diff
@@ -255,9 +268,9 @@ class Evaluation
     Evaluation operator-() const
     {
         Evaluation result;
-        result.value_ = -this->value_;
-        for (unsigned varIdx = 0; varIdx < size; ++varIdx)
-            result.derivatives_[varIdx] = - this->derivatives_[varIdx];
+        // set value and derivatives to negative
```

Comment:
since these are `Dune::FieldVector`s, is it possible to use `data_ = - other.data_`. (I know, the machine code should be identical, but it would be slightly less clutter here.)